docs/get-started/lotus/send-and-receive-fil.md

```diff
@@ -80,7 +80,7 @@ Keep your wallets' private keys safe!
 You can export and re-import a wallet, potentially to a different Lotus node, with:
 
 ```bash
-lotus wallet export <address> > wallet.private
+lotus wallet export <address> wallet.private
```

Comment:
Actually this is correct, we are redirecting the output of the command to the file.

If anything we can go with:

```suggestion
lotus wallet export "<address>" > wallet.private
```

Reply:
Ah, probably not worth it to break our example syntax patterns. I'll close this PR, we can revisit if people report problems using this.